index.html

```diff
@@ -10481,7 +10481,7 @@ <h3>User Prompt Handler</h3>
 
 <p>The <dfn>valid prompt types</dfn> are «"<code>alert</code>",
 "<code>beforeUnload</code>", "<code>confirm</code>",
-"<code>default</code>", "<code>prompt</code>"».
+"<code>default</code>", "<code>fileDialog</code>", "<code>prompt</code>"».
```

Comment:
nit: I think we should call it `file` or `filePicker` because it's a type of a prompt, and a `FileDialog` prompt sounds redundant since the prompt is already a sort of a dialog.

Reply:
done